src/docs/test-pages/02-code-tabs.md

```diff
@@ -0,0 +1,154 @@
+---
```

Comment:
Test pages for features of vuepress that are currently not in use but will be useful in the future of more docs changes going forward. These pages are excluded from the build process (see `.vuepress/config.js`)